src/controllers/update/update.filing.date.controller.ts

```diff
@@ -1,39 +1,23 @@
 import { NextFunction, Request, Response } from "express";
 import { Session } from "@companieshouse/node-session-handler";
 
-import { createAndLogErrorRequest, logger } from "../../utils/logger";
+import { logger } from "../../utils/logger";
 import * as config from "../../config";
 import { isActiveFeature } from "../../utils/feature.flag";
-import { getApplicationData, setExtraData, mapFieldsToDataObject, mapDataObjectToFields, addPrivateOverseasEntityDetailsToAppData } from "../../utils/application.data";
+import { getApplicationData, setExtraData, mapFieldsToDataObject, mapDataObjectToFields } from "../../utils/application.data";
 import { postTransaction } from "../../service/transaction.service";
 import { createOverseasEntity, updateOverseasEntity } from "../../service/overseas.entities.service";
-import { OverseasEntityKey, Transactionkey, InputDateKeys, EntityNumberKey } from '../../model/data.types.model';
+import { OverseasEntityKey, Transactionkey, InputDateKeys } from '../../model/data.types.model';
 import { FilingDateKey, FilingDateKeys } from '../../model/date.model';
 import { ApplicationData } from "../../model/application.model";
-import { getPrivateOeDetails, hasRetrievedPrivateOeDetails } from "../../service/private.overseas.entity.details";
 
-const fetchAndSavePrivateOeData = async (req: Request, appData: ApplicationData): Promise<void> => {
```

Comment:
possibly we could have keep this in.

Reply:
It would've been unused and caused a code smell. It was easier to remove it, plus we've got the commit history 🤷‍♂️